combat-trainer.lic

```diff
@@ -654,8 +689,9 @@ class SpellProcess
     @before.each { |action| bput(action['message'], action['matches']) } if @before
 
     @custom_cast = "cast my #{game_state.weapon_name}" if game_state.casting_ignite
+    @custom_cast = "cast #{game_state.healing_target}" if game_state.casting_consume
```

Comment:
Is it desirable to delay selection of the `healing_target` until the last possible moment (i.e. here)?